js/client.js

```diff
@@ -1158,6 +1162,15 @@ function toId() {
 				break;
 			}
 		},
+		saveIgnore: function () {
+			var buf = {};
+			if (!this.ignore) return; // ??
+			for (var k in this.ignore) {
+				// we set this to false so they're notified once per session of it
+				buf[k] = {notified: false};
+			}
+			app.saveIgnore();
+		},
```

Comment:
![image](https://user-images.githubusercontent.com/551184/115798475-2d41ca00-a38b-11eb-8ec4-cca63b2a8c20.png)

You forgot to actually write `saveIgnore`

Reply:
Oops. Fixed.